*** 50-synaptics.conf.orig	2014-08-12 19:56:58.488513737 +0900
--- 50-synaptics.conf	2014-08-12 20:09:35.568516385 +0900
***************
*** 18,23 ****
--- 18,25 ----
  # enabled by default. See the following link for details:
  # http://who-t.blogspot.com/2010/11/how-to-ignore-configuration-errors.html
          MatchDevicePath "/dev/input/event*"
+ 	Option "CircularScrolling" "on"
+ 	Option "corcScrollTrigger" "0"
  EndSection
  
  Section "InputClass"
